src/java.management.rmi/share/classes/javax/management/remote/rmi/RMIConnectionImpl.java

```diff
@@ -208,7 +206,11 @@ public ObjectInstance createMBean(String className,
                 throw (NotCompliantMBeanException) e;
             if (e instanceof IOException)
                 throw (IOException) e;
-            throw newIOException("Got unexpected server exception: " + e, e);
+            if (e instanceof SecurityException)
+                throw (SecurityException) e;
+            if (e instanceof RuntimeException)
+                throw (RuntimeException) e;
```

Comment:
Throughout all these changes: SecurityException is  a RuntimeException, so you do not need to check for both.

An alternative is to:
```
} catch (RuntimeException r) {
    throw r;
} catch (Exception e) {
    ....
}
```

Reply:
OK sure, I was just making it super explicit.  Can update.

Reply:
Updated - re-ran all of the management tests, looks good.